modules/frontend/metrics_query_range_handler_test.go

```diff
@@ -124,6 +125,7 @@ func TestQueryRangeHandlerRespectsSamplingRate(t *testing.T) {
 						Value:       1,
 					},
 				},
+				Exemplars: make([]tempopb.Exemplar, 0),
```

Comment:
Is this a symptom that the exemplar slice is alloced when it shouldn't be?  If it remained nil when not used that seems like small but positive change.

Reply:
Fixed!